README.md

```diff
@@ -33,6 +33,8 @@ Then you can try to connect to the endlessh server. Your SSH client should hang
 ssh -p 2222 localhost
 ```
 
+The default container user has uid/gid 2000.
```

Comment:
I feel this is irreverent here.
Why we need it here? 
If we have to keep it, it should go to line 28.